### PR TITLE
Parsers-V2: Refactor YuNet Parser

### PR DESCRIPTION
**Pull request by jkbmrz**
This PR adds a parsers-v2 refactor for `YuNetParser`, and adjusts `ImgDetectionExtended` message to accept the newly added `Keypoints` object and is also up-to-date with the newly planned changes that will get written in DAI. Detection message creator (`create_detection_message`) is also adjusted and now accepts only numpy arrays (this change also affects `MPPalmDetectionParser`, `SCRFDParser`, and `YOLOExtendedParser` which are not fixed in this PR).

NOTE: The PR also includes some pre-commit format fixing for the Regression parser and message.

### CODE REVIEW

**Comment by jkbmrz** [comment]
One note - we plan to expand `ImgDetections` in DAI to include `x_center`, `y_center`, `height`, `width`, `angle`, `confidence`, `keypoints`, and `mask`. This PR doesn't add `x_center`, `y_center`, `height`, `width` parameters because `xmin`, `ymin`, `xmax`, `ymax` are already implemented and would look a bit weird to have both. Will switch to xywh once DAI implementation is done.

File: depthai_nodes/ml/parsers/__init__.py [hunk 1]
```diff
@@ -12,7 +12,6 @@
 from .mlsd import MLSDParser
 from .parser_generator import ParserGenerator
 from .ppdet import PPTextDetectionParser
-from .classification_sequence import ClassificationSequenceParser
```
Comment (kkeroo) [inline depthai_nodes/ml/parsers/__init__.py:15]: Why removing this?

**Review by kkeroo — APPROVED** [review]
LGTM, only one comment

File: depthai_nodes/ml/parsers/__init__.py (hunk 1 above)
Comment (jkbmrz) [reply depthai_nodes/ml/parsers/__init__.py:15]: `from .classification_sequence import ClassificationSequenceParser` is present in line 3. I think this is a consequence of rebranching.

Comment (kkeroo) [reply depthai_nodes/ml/parsers/__init__.py:15]: Ah okay, I missed that.

**Pull request merged**